layouts/quickstart/index.html

```diff
@@ -0,0 +1,217 @@
+
+        </div>
+      </div>
+
+      <div class="usa-content bgcolor-ltbluegrey">
+        <div class="usa-grid">
+          <section class="step first">
+            <h2>Get trial access and a free sandbox account</h2>
+            <p>
+              Federal employees and contractors with an email ending in .mil, .gov, or .fed.us can obtain a free sandbox account. To register, you’ll need access to your email and a smartphone with Google Authenticator installed.
+            </p>
+            <!-- <form>
+              <label for="email">Federal email address</label>
+              <div class="form-wrapper">
+                <input type="text" name="email" value="">
+                <input type="submit" value="Submit">
+              </div>
+            </form> -->
+            <p>
+              <a href="https://cloud.gov/docs/getting-started/accounts/" class="usa-button">Get access</a>
+            </p>
+          </section>
+        </div>
+      </div>
+
+      <div class="usa-content bgcolor-ltputty">
+        <div class="usa-grid">
+          <section class="step second">
+            <h2>Go build in your sandbox</h2>
+            <p>
+              Follow any of the tutorials below to launch your own web application from your sandbox. You’ll need your cloud.gov account and some familiarity with Windows Powershell or Mac/Linux terminal.
+            </p>
+
+            <div class="content-resources">
+              <section>
+                <h3>
+                  <span class="eyebrow">Tutorial</span>
+                  Watch our video tutorials
+                </h3>
+                <p>
+                  Get an introduction to basic cloud.gov concepts and learn how to run applications in your sandbox.
+                </p>
+                <div class="">
+                  <article class="usa-width-one-third">
+                    <iframe width="280" height="157" src="https://www.youtube.com/embed/LKb0liZyepA?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
+                    <!-- <iframe width="208" height="117" src="https://www.youtube.com/embed/LKb0liZyepA?rel=0" frameborder="0" allowfullscreen></iframe> -->
+                    <h4>Part 1: Why cloud.gov?</h4>
+                    <p class="duration">
+                      27 minutes
+                    </p>
+                    <p>
+                      Get an overview of the cloud.gov platform and how it reduces lead time, and  ensures security and compliance.
+                    </p>
+                    <ul>
+                      <li>
+                        <a href="../assets/files/01-slides-why-cloudgov.pdf">Download slides</a>
+                      </li>
+                    </ul>
+
+                  </article>
+                  <article class="usa-width-one-third">
+                    <iframe width="280" height="157" src="https://www.youtube.com/embed/ip8G_Rafb_c?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
```

Comment:
Link to 

https://www.youtube.com/embed/ip8G_Rafb_c?rel=0&t=3m1s

Since the first 3 minutes are not relevant